bin/bampe_rm_orphan.py

```diff
@@ -17,17 +17,25 @@
 ############################################
 ############################################
 
-Description = 'Remove singleton reads from paired-end BAM file i.e if read1 is present in BAM file without read 2 and vice versa.'
+Description = (
+    "Remove singleton reads from paired-end BAM file i.e if read1 is present in BAM file without read 2 and vice versa."
+)
```

Comment:
Is this linted with Black?

Reply:
Yep, tests seem to be passing for that on the PR.